lib/Installation/SystemRole/SystemRolePage.pm

```diff
@@ -0,0 +1,48 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: The module provides interface to act on System Role page in
+#          the installer.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package Installation::SystemRole::SystemRolePage;
+use strict;
+use warnings;
+
+sub new {
+    my ($class, $args) = @_;
+    my $self = bless {
+        app => $args->{app}
+    }, $class;
+    return $self->init();
+}
+
+sub init {
+    my $self = shift;
+    $self->{sis_role} = $self->{app}->singleitemselector({id => 'role_selector'});
```

Comment:
Shall it be `sys_role`?

Reply:
I was using the first three letters from the control name to prepend the name, now I chose another three letter 'sel' from selector.